src/signer_node/mod.rs

```diff
@@ -643,53 +643,6 @@ mod tests {
         (node, broadcaster)
     }
 
-    #[test]
-    fn test_pubkey_list_sort() {
-        use bitcoin::util::key::PublicKey;
-        use std::str::FromStr;
-
-        let pubkey_list = vec![
-            PublicKey::from_str(
-                "03831a69b8009833ab5b0326012eaf489bfea35a7321b1ca15b11d88131423fafc",
-            )
-            .unwrap(),
-            PublicKey::from_str(
-                "02ce7edc292d7b747fab2f23584bbafaffde5c8ff17cf689969614441e0527b900",
-            )
-            .unwrap(),
-            PublicKey::from_str(
-                "02785a891f323acd6cef0fc509bb14304410595914267c50467e51c87142acbb5e",
-            )
-            .unwrap(),
-            PublicKey::from_str(
-                "02d111519ba1f3013a7a613ecdcc17f4d53fbcb558b70404b5fb0c84ebb90a8d3c",
-            )
-            .unwrap(),
-            PublicKey::from_str(
-                "02472012cf49fca573ca1f63deafe59df842f0bbe77e9ac7e67b211bb074b72506",
-            )
-            .unwrap(),
-        ];
-        let threshold = 3;
-        let private_key = TEST_KEYS.key[0];
-        let to_address = address(&private_key);
-
-        let params = NodeParameters::new(
-            to_address,
-            pubkey_list.clone(),
-            private_key,
-            threshold,
-            MockRpc {
-                return_block: safety_error("Not set block.".to_string()),
```

Comment:
After this change, `cargo test` report the warning like this:

```
warning: unused import: `safety_error`
   --> src/signer_node/mod.rs:525:37
    |
525 |     use crate::rpc::tests::{safety, safety_error, MockRpc};
    |                                     ^^^^^^^^^^^^
    |
    = note: `#[warn(unused_imports)]` on by default

```

Reply:
Thanks, I'll fix it.

Reply:
updated 8b22c1c66b30947f07262a75c9a6da8a874a8ced